elastic/kibana
# [Fleet] added a logic to use `destinationId` when tagging imported SOs

## Summary

Fixes https://github.com/elastic/kibana/issues/147364

When installing an integration, and some assets already exist in Kibana, the imported SOs will recieve a newly generated ID. This caused a problem previously when the tagging logic didn't find the SOs with their original ID.
The fix takes the new `destinationId` of those SOs, so the tagging can complete successfully.

**Steps to verify**
1. Add system integration in default space
2. Enroll an agent and wait for system metrics to be ingested
3. Create an Admin space
4. Remove the system package policy, and uninstall the system package from Default space
5. Go to Admin space and add system integration again
6. Verify that the integration is added successfully, assets are created in Admin space and assets are tagged with Managed and System tags.


### Checklist
